src/averageAges.js

```diff
@@ -15,47 +15,62 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let males = people.filter(({ sex }) => sex === 'm');
```

Comment:
Part of this line has code that is repeated. Could you possibly turn it into a variable or a global variable?

`const isMale = (person) => person.sex === 'm';`
this can be used as a global variable outside any particular function, so that you can use it everywhere